packages/components/src/Layout/Semantics/Story/Semantics.story.tsx

```diff
@@ -25,18 +25,65 @@
  */
 
 import React from 'react'
-import {
-  Layout,
-  Header,
-  Aside,
-  Footer,
-  Grid,
-  Section,
-} from '@looker/components'
+import { Story } from '@storybook/react/types-6-0'
 import styled from 'styled-components'
+import { Layout, Header, Aside, AsideProps, Footer, Section } from '..'
+import { Grid } from '../../Grid'
 
 export default {
-  title: 'Layout/Semantics',
+  title: 'Layout',
+}
+
+const Template: Story<AsideProps> = (args) => (
+  <Aside p="large" text-align="center" {...args} />
+)
+
+export const DefaultColor = Template.bind({})
+DefaultColor.args = {
+  border: true,
+  children: 'Aside',
+}
```

Comment:
Super-minor nit (only noticed because it was reading through the snapshots above and it read really nicely until this one) :)

```suggestion
export const DefaultBorderColor = Template.bind({})
DefaultBorderColor.args = {
  border: true,
  children: 'Aside',
}
```